diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/utils/indentation/Checkers.kt

```diff
@@ -174,13 +176,25 @@ internal class DotCallChecker(config: IndentationConfig) : CustomIndentationChec
     private fun ASTNode.isDotBeforeCallOrReference() = elementType.let { it == DOT || it == SAFE_ACCESS } &&
             treeNext.elementType.let { it == CALL_EXPRESSION || it == REFERENCE_EXPRESSION }
 
+    private fun ASTNode.isCommentBeforeDot() : Boolean {
+        if (elementType == EOL_COMMENT || elementType == BLOCK_COMMENT) {
+            var nextNode = treeNext
+            while (nextNode != null && (nextNode.elementType == WHITE_SPACE || nextNode.elementType == EOL_COMMENT)) {
+                nextNode = nextNode.treeNext
+            }
+            return nextNode.elementType == DOT
```

Comment:
Maybe, better to use `nextNode.isDotBeforeCallOrReference()` here?

Reply:
Yeah, that seems better